fe/fe-core/src/test/java/org/apache/doris/qe/StmtExecutorTest.java

```diff
@@ -731,5 +733,18 @@ public void testUseFail(@Mocked UseStmt useStmt, @Mocked SqlParser parser) throw
 
         Assert.assertEquals(QueryState.MysqlStateType.ERR, state.getStateType());
     }
+
+    @Test(expected = AnalysisException.class)
```

Comment:
The unit test is too few...
where is the create/alter/drop/show sql block rule unit test?

Reply:
I will submit it later